bb_wrapper/models/pagamentos.py

```diff
@@ -167,138 +167,201 @@ class LiberarPagamentos(BaseModel):
+    65: "Código de Identificação do Tributo não informado",
+    66: "Período de apuração não informado",
+    67: "Numero de Referência não informado",
+    68: "Valor Principal não é numérico",
+    69: "Valor Principal não informado",
+    70: "Valor da Multa não é numérico",
+    71: "Valor dos Juros/Encargos não é numérico",
+    72: "Data de Vencimento não informada",
+    73: "Mês e ano de competência não informados",
+    74: "Valor previsto do pagamento do INSS não é numérico",
+    75: "Valor previsto do pagamento do INSS não informado",
+    76: "Valor de Outras Entidades não é numérico",
+    77: "Valor de Atualização Monetária não é numérico",
+    78: "Valor de Atualização Monetária não é numérico",
+    79: "Período de apuração inválido",
+    80: "Conta de crédito inválida. Informe o numero   sem o 45 do início.",
+    81: "A conta informada não pertence ao funcionário.",
+    82: "Pagamento permitido apenas para pessoas físicas.",
+    83: "Agência e Conta incorretos.",
+    84: "A conta informada não está ativa.",
+    85: "Conta não permite crédito de salário. Informe outra conta.",
+    86: "Ambos os campos agência de crédito e conta pagamento foram informados",
+    90: "Mês de competência inválido",
+    91: "Valor de outras deduções inválido",
+    92: "Valor de outros acréscimos inválido",
+    93: "Código da forma de identificação do cliente não foi informado",
+    94: "DDD do cliente do Pix não foi informado",
+    95: "Telefone do Cliente do Pix não foi informado",
+    96: "Email do cliente do Pix não foi informado",
+    97: "Chave Aleatória do Cliente do Pix não foi informado",
+    98: "Código de tipo de conta do Cliente do Pix não foi informado",
     99: "Consultar o Banco para detalhar o erro",
-    200: "Insuficiência de Fundos: Débito Não Efetuado",
+    100: "E-mail inválido",
+    101: "Email do cliente do PIX não deve conter caractere especial",
+    102: "Telefone Inválido",
+    103: "DDD inválido",
+    104: "E-mail com tamanho maior que 77 caracteres.",
+    105: "Conta de crédito inválida. Informe um número de conta válido.",
+    106: "CPF inválido. Informe um CPF válido.",
+    107: "CNPJ inválido. Informe um CNPJ válido.",
+    108: "Número do documento de crédito inválido. Informe um número válido.",
+    109: "Número do documento de débito inválido. Informe um número válido.",
+    110: "Valor do pagamento inválido. Informe um valor válido.",
+    111: "Valor nominal inválido. Informe um valor válido.",
+    112: "Valor de desconto inválido. Informe um valor válido.",
+    113: "Valor de mora inválido. Informe um valor válido.",
+    114: "Número de inscrição do beneficiário inválido. Informe um número válido.",
+    115: "Número de inscrição do pagador inválido. Informe um número válido.",
+    116: "Número de inscrição do avalista inválido. Informe um número válido.",
+    117: "Nº de identifiação do contribuinte DARF inválido. Informe um número válido.",
```

Comment:
Acho que seria bom padronizar o uso de "Número" ao invés de "Nº"
```suggestion
    117: "Número de identifiação do contribuinte DARF inválido. Informe um número válido.",
```